packages/fast-components-styles-msft/src/design-system/index.ts

```diff
@@ -16,22 +16,27 @@ export type DensityOffset = -3 | -2 | -1 | 0 | 1 | 2 | 3;
 
 export interface DesignSystem {
     /**
-     * The value typically used for backgrounds of elements
+     * The background color of the current context.
+     * May be used to draw an actual background or not. Either way, any color recipes evaluated within this context will use this as their basis.
      */
     backgroundColor: string;
 
     /**
-     * The accent color, which the accentPalette is based on
+     * The accent color, which the accent palette is based on.
+     * Keep this value in sync with accentPalette.
      */
     accentBaseColor: string;
 
     /**
-     * Configuration object to derive the neutral palette. Expects a ColorPaletteConfig from @microsoft/fast-colors
+     * An array of colors in a ramp from light to dark, used to look up values for neutral color recipes.
+     * Typically generated using ComponentStateColorPalette in @microsoft/fast-colors.

```

Comment:
Unless we’re going to detail non-typical ways, can we just expressly detail what we are driving to our implementation?